crates/interledger-packet/src/packet.rs

```diff
@@ -562,7 +568,20 @@ fn deserialize_envelope(
         before - peek.len()
     };
 
-    let content = reader.peek_var_octet_string()?;
+    let content = reader.read_var_octet_string()?;
+
+    #[cfg(fuzzing)]
```

Comment:
I know I suggested this `cfg(fuzzing)` but we do additionally have the `strict` feature which might be useful in the future. However ok to leave this as is for now, perhaps if other crates find the need for this functionality, then it needs to be turned into feature.

```suggestion
    // TODO: consider moving behind #[cfg(feature = "strict")] for dependent crate benefit
    #[cfg(fuzzing)]
```